eng/pipelines/templates/stages/archetype-sdk-tests-isolated.yml

```diff
@@ -44,6 +44,12 @@ parameters:
 - name: Location
   type: string
   default: ''
+- name: PreGenerationSteps
```

Comment:
What are these steps used for?

Reply:
It's a prerequisite to get the dynamic configuration before generating jobs. It's used to update the spring supported versions for matrix config file, such as `sdk/spring/pipeline/supported-version-matrix.json`, and `sdk/spring/pipeline/cosmos-integration-matrix.json`.